spec/features/delete_user_spec.rb

```diff
@@ -0,0 +1,15 @@
+require 'spec_helper'
+
+describe 'Deleting a User' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.